submissions/build.gradle.kts

```diff
@@ -4,7 +4,7 @@ plugins {
     id("org.springframework.boot") version "3.3.2"
```

Comment:
3.3.3 is a security fix (https://github.com/CDCgov/prime-reportstream/pull/15695).  We should probably take /submissions PRs out of this and handle them separately.

Reply:
We may want to split out /shared PRs as well.